### DISCUSSION TITLE
Horizontal mouse scrolling

### DISCUSSION THREAD

**Discussion by jmunckhof** [asmyshlyaev177/react-horizontal-scrolling-menu#173] [Q&A]
Hello,

I have a question about horizontal scrolling with mouse wheel, I have read the documentation and I can't find it.
How can I let users horizontal scroll when they are scrolling with their mouse?

**Comment by asmyshlyaev177** [comment]
Here example https://codesandbox.io/s/react-horizontal-scrolling-menu-v2-basic-example-swg0y?file=/src/index.tsx
and this https://codesandbox.io/s/react-horizontal-scrolling-menu-v2-drag-by-mouse-o3u2t?file=/src/index.tsx

**Reply by jmunckhof** [reply]
Thankyou, It feels a little buggy, same for the arrows. Is there a way to scroll smoothly?

**Reply by asmyshlyaev177** [reply]
Just standard browser scroll, what do you mean by buggy?
